webapp/tests/karma/ts/services/contact-summary.service.spec.ts

```diff
@@ -25,6 +25,13 @@ describe('ContactSummary service', () => {
       getHomeVisitStats: sinon.stub(),
       getUHCInterval: sinon.stub()
     };
+    chtScriptApi = {
+      v1: { hasPermission: () => {} }
```

Comment:
Please prefer to use `sinon.stub()` instead of noop functions.

Reply:
Fixed!